site/documentation.html

```diff
@@ -149,6 +149,7 @@ <h2><span class="text-capitalize">Apache Spark<span class="tm">&trade;</span></s
 <p>Setup instructions, programming guides, and other documentation are available for each stable version of Spark below:</p>
 
 <ul>
+  <li><a href="/docs/3.2.0/">Spark 3.2.1</a></li>
```

Comment:
(Should be 3.2.1)

Reply:
I think this should be generated by `documentation.md`? @huaxingao do you manually modify this file?

Reply:
I had the wrong version in `documentation.md`. I fixed it but forgot to run `jekyll build`.